src/gui/screen_lan_settings.c

```diff
@@ -5,45 +5,44 @@
  *      Author: Migi
  */
 
-//#define STATIC_SAVE_LOAD_CONFIG
-
 #include "screen_lan_settings.h"
 #include "lwip/dhcp.h"
 #include "lwip/netifapi.h"
 #include "lwip.h"
 #include <stdlib.h>
 #include <stdbool.h>
-#ifdef STATIC_SAVE_LOAD_CONFIG
-    #include "ini.h"
-    #include "ff.h"
-#endif //STATIC_SAVE_LOAD_CONFIG
+#include "ini.h"
+#include "ff.h"
 #include <string.h>
 
 #define MAC_ADDR_START    0x1FFF781A //MM:MM:MM:SS:SS:SS
 #define MAC_ADDR_SIZE     6
 #define MAX_INI_SIZE      100
 #define IP4_ADDR_STR_SIZE 16
 
+#define _change_static_to_static() _change_dhcp_to_static()
+
 typedef enum {
     MI_RETURN,
-#ifdef STATIC_SAVE_LOAD_CONFIG
+    MI_SWITCH,
+    MI_TYPE,
     MI_SAVE,
     MI_LOAD,
-#endif //STATIC_SAVE_LOAD_CONFIG
 } MI_t;
 
 static char *plan_str = NULL;
+static bool conn_flg = false; // wait for dhcp to supply addresses
 static networkconfig_t config;
-#ifdef STATIC_SAVE_LOAD_CONFIG
 static const char ini_file_name[] = "/lan_settings.ini"; //change -> change msgboxes
+static const char *LAN_switch_opt[] = { "On", "Off", NULL };
+static const char *LAN_type_opt[] = { "dhcp", "static", NULL };
 static char ini_file_str[MAX_INI_SIZE];
 extern bool media_is_inserted();
-#endif //STATIC_SAVE_LOAD_CONFIG
 const menu_item_t _menu_lan_items[] = {
-#ifdef STATIC_SAVE_LOAD_CONFIG
+    { { "LAN", 0, WI_SWITCH, .wi_switch_select = { 0, LAN_switch_opt } }, SCREEN_MENU_NO_SCREEN },
+    { { "LAN type", 0, WI_SWITCH, .wi_switch_select = { 0, LAN_type_opt } }, SCREEN_MENU_NO_SCREEN },
```

Comment:
What is`LAN type`? 😳

Maybe `LAN IP`, so it shows as 
```
LAN IP            static/dhcp
```
 on the screen?

Reply:
@michalrudolf please rename to LAN IP